loadtest/user/userentity/actions.go

```diff
@@ -974,10 +974,10 @@ func (ue *UserEntity) AutocompleteUsersInChannel(teamId, channelId, username str
 	}
 	usersMap := make(map[string]bool, len(users.Users)+len(users.OutOfChannel))
 	for _, u := range users.Users {
-		usersMap[u.Username] = true
+		usersMap[u.Id] = true
 	}
 	for _, u := range users.OutOfChannel {
-		usersMap[u.Username] = false
+		usersMap[u.Id] = false
```

Comment:
Oh good catches!

Reply:
Do these change have any practical differences? Aren't username unique as well?

Oh I guess they are needed now to be able to feed the ids to `getProfileImageForUsers` but it was working fine before.